tools/install.py

```diff
@@ -132,8 +133,7 @@ def files(action):
       # in its source - see the _InstallableTargetInstallPath function.
       if sys.platform != 'darwin':
         output_prefix += 'lib.target/'
-
-  action([output_prefix + output_file], 'bin/' + output_file)
+    action([output_prefix + output_file], 'bin/' + output_file)
```

Comment:
how about:

```python
action([output_prefix + output_file], ('bin/' if 'false' == variables.get('node_shared') else 'lib/) + output_file)
```

that only affect one line.

Reply:
Nice suggestion! Merged to one statement but still 2 lines because of the 80 chars per line rule :)